library/photon

```diff
@@ -1,14 +1,74 @@
-# this file is generated via https://github.com/vmware/photon-docker-image/blob/b92a2b946e9f8237daf41ec0cd372d9a05264bf5/generate-stackbrew-library.sh
+# this file is generated via https://github.com/vmware/photon-docker-image/blob/574c983eb8397934324e0f8ca5824d8e00e15462/generate-stackbrew-library.sh
 
 Maintainers: Fabio Rapposelli <fabio@vmware.com> (@frapposelli),
              Alexey Makhalov <amakhalov@vmware.com> (@YustasSwamp)
 GitRepo: https://github.com/vmware/photon-docker-image.git
 Directory: docker
 
-Tags: 2.0, 2.0-20180704, latest
+Tags: 2.0-20180704, 2.0, latest
 GitFetch: refs/heads/2.0-20180704
 GitCommit: e6b78e76f36da84a67c5b8e411a50514a1b62357
 
-Tags: 1.0, 1.0-20180703
+Tags: 2.0-20180703
```

Comment:
Any particular reason to add all the old versions listed here?  Removing it here will remove it from the "Supported" section on the Hub readme, but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
oh, sorry. let me check. Looks like script make it mess a little.